src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -645,9 +654,12 @@ export const selectHopSellAccountId = createSelector(
 
 export const selectHopExecutionMetadata = createDeepEqualOutputSelector(
   selectTradeQuoteSlice,
-  (_state: ReduxState, hopIndex: number) => hopIndex,
-  (swappers, hopIndex) => {
-    return hopIndex === 0 ? swappers.tradeExecution.firstHop : swappers.tradeExecution.secondHop
+  (_state: ReduxState, tradeId: TradeQuote['id']) => tradeId,
```

Comment:
From what I can see execution-wise, we always use `activeQuote.id` as a `tradeId` - do we want to `selectActiveQuoteId` here instead of passing it as a param, or could we possibly pass something different (i.e `confirmedQuote.id`) eventually and prefer to keep it that way?

Reply:
Closing as we want to keep things declarative, avoid tsc hell, and keep the door open for using a different `tradeId` in the future - more context in https://github.com/shapeshift/web/pull/7551#issuecomment-2290034322